core/db.py

```
@@ -294,3 +295,121 @@ async def init_db_async() -> None:
+def get_db_connection():
+    """Get a database connection for testing purposes.
+
+    RU: Возвращает соединение c базой данных для тестирования.
+    EN: Returns a database connection for testing purposes.
+    """
+    try:
+        return _RAW_ENGINE.connect()
+    except Exception as e:
+        import logging
+
+        logging.error(f"Failed to get database connection: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)